src/sinks/clickhouse/mod.rs

```diff
@@ -1,5 +1,9 @@
 mod config;
+mod convert;
 mod http_sink;
+mod native_sink;
+mod native_service;
+mod parse;
+pub use self::config::*;
 #[cfg(all(test, feature = "clickhouse-integration-tests"))]
-mod integration_tests;
-pub use self::config::ClickhouseConfig;
+mod integration_tests;
```

Comment:
Missing a newline at the EOF, same for the other files, perhaps a configuration with your editor ?